### PR TITLE
Items from Jeffrey Yasskin's review

### PR DESCRIPTION
**Pull request by bc-pi**
which can be found at #389

### CODE REVIEW

**Comment by Sakurann** [comment]
I think the following change might be cleaner per `"Check that the Key Binding JWT is valid in all other respects" should call a particular algorithm from the other specs that it cites.` in Jefferey's review.

"9. Check that the Key Binding JWT is valid in all other respects, per [@!RFC7519] and [@!RFC8725]."
-> 
"9. Check that the Key Binding JWT is a valid JWT in all other respects, per [@!RFC7519] and [@!RFC8725]."

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 1]
```diff
@@ -345,13 +345,15 @@ minimum length of the salt, and choice of a hash algorithm.
 
 If the Issuer wants to enable Key Binding, it includes a public key
 associated with the Holder, or a reference thereto.
+The `jwk` member of the `cnf` claim as defined in Section 3.2 of [@!RFC7800] is
+RECOMMENDED for doing so, however, other means MAY be used.
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:349]: as Brian said [here](https://github.com/oauth-wg/oauth-selective-disclosure-jwt/issues/389#issuecomment-1858648845)

> This draft intentionally doesn't prescribe how keys are distributed, identified, selected, trusted, etc. any more than JWT/JWS do (e.g. https://datatracker.ietf.org/doc/html/rfc7515#section-6, https://datatracker.ietf.org/doc/html/rfc7515#appendix-D, etc) so I think this is redundant and using `cnf` in examples is sufficient.

```suggestion
```

**Comment by bc-pi** [comment]
> I think the following change might be cleaner per `"Check that the Key Binding JWT is valid in all other respects" should call a particular algorithm from the other specs that it cites.` in Jefferey's review.
> 
> "9. Check that the Key Binding JWT is valid in all other respects, per [@!RFC7519] and [@!RFC8725]." -> "9. Check that the Key Binding JWT is a valid JWT in all other respects, per [@!RFC7519] and [@!RFC8725]."

Sure, that's done in 4442c3a06767498ee24154024d8dff987325b08f

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 1 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:349]: I believe this is a useful area to provide a little more guidance that was previously lacking.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 2]
```diff
@@ -1709,6 +1712,13 @@ data. The original JSON data is then used by the application. See
 
    [[ To be removed from the final specification ]]
 
+   -08
+
+* Make RFCs 0020 and 7515 normative references
+* Be a bit more prescriptive in suggesting RFC7800 cnf/jwk be used to convey the Key Binding key
+* Editorial changes aimed at improved clarity
```
Comment (danielfett) [inline draft-ietf-oauth-selective-disclosure-jwt.md:1719]: This looks like it needs more indentation.

**Review by danielfett — APPROVED** [review]
Thanks for creating the PR!

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 2 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1719]: The resultant output is okay https://drafts.oauth.net/oauth-selective-disclosure-jwt/yasskin-asking/draft-ietf-oauth-selective-disclosure-jwt.html#name-document-history

Comment (danielfett) [reply draft-ietf-oauth-selective-disclosure-jwt.md:1719]: I didn't expect that, but in that case this is not an issue, thanks!

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 1 above)
Comment (bc-pi) [inline draft-ietf-oauth-selective-disclosure-jwt.md:349]:
```suggestion
The `jwk` member of the `cnf` claim as defined in Section 3.2 of [@!RFC7800] is
suggested for doing so, however, other means can be used.
```

Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md]: The text with that guidance has been changed to not use rfc2119 uppercase key words.

File: draft-ietf-oauth-selective-disclosure-jwt.md [hunk 3]
```diff
@@ -345,13 +345,15 @@ minimum length of the salt, and choice of a hash algorithm.
...
+The `jwk` member of the `cnf` claim as defined in Section 3.2 of [@!RFC7800] is
+suggested for doing so, however, other means can be used.
 
 It is out of the scope of this document to describe how the Holder key pair is
 established. For example, the Holder MAY create a key pair and provide a public key to the Issuer,
 the Issuer MAY create the key pair for the Holder, or
 Holder and Issuer MAY use pre-established key material.
 
-Note: The examples in this document use the `cnf` claim defined in [@RFC7800] to include
+Note: The examples throughout this document use the `cnf` claim to include
```
Comment (Sakurann) [inline draft-ietf-oauth-selective-disclosure-jwt.md:356]:
```suggestion
Note: The examples throughout this document use the `cnf` claim ([@RFC7800]) to include
```
was there anything wrong with referencing RFC7800..?

**Review by Sakurann — APPROVED** [review]
one nit, otherwise looks good - thank you, Brian!

File: draft-ietf-oauth-selective-disclosure-jwt.md (hunk 3 above)
Comment (bc-pi) [reply draft-ietf-oauth-selective-disclosure-jwt.md:356]: "fine"

**Pull request merged**